docs/user-guide/api-mediation/api-gateway-configuration.md

```diff
@@ -1,33 +1,51 @@
 # Advanced Gateway features configuration
 
-As a system programmer who wants to configure advanced Gateway features of the API Mediation Layer, set the following parameters by modifying the `<Zowe install directory>/components/api-mediation/bin/start.sh` file. The parameters begin with the `-D` prefix, similar to all the other parameters in the file. Zowe needs to be restarted for these changes to take effect.
+As a system programmer who wants to configure advanced Gateway features of the API Mediation Layer, set the following parameters by modifying either:
+- `<Zowe install directory>/components/api-mediation/bin/start.sh` file. The parameters begin with the `-D` prefix, similar to all the other parameters in the file. Zowe needs to be restarted for these changes to take effect.
```

Comment:
I think something is missing here. You say, "by modifying either", but then there is only one shell.

Reply:
Please check this meta-description for accuracy.

Reply:
Thanks for the catch! the current file lists two items: 
- `<Zowe install directory>/components/api-mediation/bin/start.sh` 
- `<Zowe instance directory>/instance.env`